kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -422,4 +454,19 @@ static Condition resolvedRefsUnknown(Clock clock,
         return newUnknownCondition(clock, observedGenerationSource, Condition.Type.ResolvedRefs, e);
     }
 
+    static String slug(String singular, String group, String name) {
+        return singular + "." + group + "/" + name;
+    }
+
+    static String slug(Class<? extends CustomResource> annotatedCrdClass, String crName) {
+        return slug(
+                annotatedCrdClass.getAnnotation(Singular.class).value(),
+                annotatedCrdClass.getAnnotation(Group.class).value(),
+                crName);
+    }
+
+    static String slug(CustomResource resource) {
+        return slug(resource.getSingular(), resource.getGroup(), name(resource));
+    }
```

Comment:
These are relevant to https://github.com/kroxylicious/kroxylicious/issues/1961

Reply:
just picked up #1961 :grinning: guess we'll be clashing